src/routes/v2/authenticated/index.js

```diff
@@ -27,13 +31,21 @@ const getAuthenticatedSubrouter = ({
   const netlifyTomlService = new NetlifyTomlService()
 
   const sitesV2Router = new SitesRouter({ sitesService })
+  const collaboratorsRouter = new CollaboratorsRouter({
+    collaboratorsService,
+    authorizationMiddleware,
+  })
   const usersRouter = new UsersRouter({ usersService })
   const netlifyTomlV2Router = new NetlifyTomlRouter({ netlifyTomlService })
 
   const authenticatedSubrouter = express.Router({ mergeParams: true })
 
   authenticatedSubrouter.use(authenticationMiddleware.verifyJwt)
 
+  authenticatedSubrouter.use(
+    "/sites/:siteName/collaborators",
+    collaboratorsRouter.getRouter()
+  )
```

Comment:
Should this router be part of `authenticatedSites` instead?

Reply:
The way I saw it was that `authenticatedSites` was responsible for site content, i.e. what the GitHub repo handles today, while `authenticatedSites` was responsible for site-related meta features (e.g. collaborators, review requests, notifications).

But this is definitely up for discussion. Tagging @alexanderleegs and @seaerchin for their thoughts since they are working on RR+Notifications at the moment

Reply:
I think we haven't really made a decision on this - currently the `authenticatedSubrouter` and `authenticatedSitesSubrouter` are split as such because of the params they require. I think either way works, though I'm leaning towards reusing the `authenticatedSites` router to make use of the related subrouters which go together with it, to prevent having to set up siteName retrieval and other site specific operations again.